plugins/quickbooks-online/README.md

```diff
@@ -45,10 +45,23 @@ This plugin handles import of your [UCRM](https://ucrm.ubnt.com/) customers, pay
 
 ## To be done in future version
 (Feel free to push your upgrades in this repo.)
-- Configurable date of the first payment or invoice to be imported. 
 - Remove entity from QB when the related entity is deleted in UCRM.
 
 ## Changelog
+### 1.1.5 (2021-12-01)
```

Comment:
Probably should be "2.0.0", same as in the manifest.
```suggestion
### 2.0.0 (2021-12-01)
```

Reply:
Done